src/App/tests/test_Extensions.py

```diff
@@ -0,0 +1,38 @@
+import types
+import unittest
+from pathlib import Path
+
+import App.config
+from App.Extensions import getObject
+
+
+class GetObjectTests(unittest.TestCase):
+    """Testing ..Extensions.getObject()."""
+
+    def setUp(self):
+        cfg = App.config.getConfiguration()
+        assert not hasattr(cfg, 'extensions')
+        cfg.extensions = Path(__file__).parent / 'fixtures'
+
+    def tearDown(self):
+        cfg = App.config.getConfiguration()
+        del cfg.extensions
+
+    def test_Extensions__getObject__1(self):
+        """It returns an the requested function.
```

Comment:
Something is wrong here.
I assume, the documentation should be something like:
`Check that "getObject" returns the requested function and that its code object has the path set.`

Reply:
Fixed in https://github.com/zopefoundation/Zope/pull/1138/commits/e0aff9562db2bfb623c81740ebbb538dadab0439.